data/favicon.json

```diff
@@ -339,5 +339,7 @@
 	"yelp.com": "yelp.com.ico",
 	"yoast.com": "yoast.com.png",
 	"youtube.com": "youtube.com.png",
-	"zeit.de": "zeit.de.svg"
+	"zeit.de": "zeit.de.svg",
+	"zone.ee": "zone.ee.svg",
+	"zone.fi": "zone.ee.svg"
```

Comment:
Did you have a look how that would look with the icon you include here?

Note that the favicon will only be shown as a small quadratic icon. So the same icon that is used for the website (https://www.zone.ee/static/cropped-favicon-512x512-1-32x32.png) is probably a better fit than the currently included rectangular icon.

Reply:
The choice was between following BIMI/VMC or increased legibility. I'll choose the latter knowing it is fine to do so.

Reply:
As a site note, BIMI recommends the usage of an icon with a square aspect ratio https://bimigroup.org/creating-bimi-svg-logo-files/.

Reply:
@lieser VMC on the other hand requires a valid trademark. Otherwise yes, we'd use a more square icon.